content/nomos/updates/Athens Offsite Notes.md

```diff
@@ -0,0 +1,103 @@
+
+### `Engineering`
+
+At Nomos, there are some needs that could be covered by Vac. To be more precise: understanding on what Vac provides that could be helpful, understanding what is required for Vac to start on that support, a plan/pipeline for both teams to interact. Some of the proposals:
+
+DST:
+
+- At the moment, the most prominent project is the analysis of the Nomos mixnet.
+- This will serve as the first project interaction as it is also the highest priority.
+- Ongoing conversations on what is needed are already in place.
+
+QA:
+
+- Nomos node is in high need of a testing plan. But no real bandwidth for it.
+- Vac has specialized people with the experience and the workforce.
+- Cryptarchia and NomosDA would be the focus. But this is gonna be delayed until seen how the mixnet collaboration goes.
+- As part of this **specifications** will be reviewed by the Vac team so they can be improved. Eventually specifications will be forked and served as part of Vac too.
+
+## `Comms`
+
+##### `Key Market Trends / Research
+
+Privacy Concerns:
+
+- Building our case for privacy our protection and viability of the project. We need to define the terms of privacy and create that shield. Privacy is necessary for personal items to be possible, health, business, institutions. 
+
+Network States:
+
+- Long term play. When network states become a thing we need to be positioned. Excitement right now in crypto is harder to get, but we need to be prepared.
+
+Technical Narratives:
+
+- Litenodes. Rules without ruler (Celestia). Our blockchain is a cockroach (meme)
+- Data Availability Sampling (warrants a paper)
+- Restaking (now that is a thing, we were early to it)
+
+Decentralisation 
+
+- We can be more decentralised than any other blockchain ever. 
+
+Competitor Research
+
+- Celestia - Celestia is not built for an end of the world situation.
+- Anoma
+
+##### `Technical Milestones
+
+Product Development
+
+- 1st Milestone: Dark paper release, with a potential litepaper version.
+- 2nd Milestone: Testnet
+
+##### `Growth Plan 
+
+Target Audience: 
+
+- Degens Refine for those who care and believers.
+- Institutions: Long term. Governments. Best way of protection. Africa?
+- Developers/Contributors
+- Node Operators: Long term.
+
+Movement Building (Community) - Educational Initiatives: 
+
+- Look into monad
+- Roles for incentivising culture, memes.
+- Ambassador program
+- Developer Outreach: Filip is preparing Lore, comprehensive documentation, FAQ, tool kits, content etc.
+
+Incentive Programs 
+
+- Logos NFTs
+- When we have zones we can get them to deploy during hackathons?
+
+Partnerships / Collaborations:
+
+- Web3 Privacy Now Partnership.
+
+##### `Target audience:
+
+- We need real believers. Those who understand it need to preach it. Zealots. Most people won’t understand it. 
```

Comment:
Careful with what you put here...
Separate internal notes from public notes. These things shouldn't be here. Our strategy is internal, not public.